### ISSUE TITLE
Using v-if and v-for on the same element is not recommended

### ISSUE THREAD

**Issue by Cannonb4ll** [icebob/vue-form-generator#84]
Currently getting the error in the title.

Full error:

```
vue.common.js?4a36:1137[Vue warn]: <div v-for="field in fields" v-if="fieldVisible(field)">: Using v-if and v-for on the same element is not recommended - consider filtering the source Array instead. (found in component: <vue-form-generator>)
```

Am I doing something wrong?

**Comment by lionel-bijaoui** [comment]
I don't think you did anything wrong, this is something we missed. We never got this message on our dev environment, because we were using `"vue": "1.0.26"`.
I just updated `"vue": "1.0.27"` and I got the same message. So this is new and we need to change the way it work.
You don't need to worry too much about this message for now, and we will fix it.
Thank you for your feedback

**Comment by Cannonb4ll** [comment]
Then I experienced the same issue, I did an npm update yesterday and with that some new versions came along. Ever since this warning occured.

Application works just fine though, will be waiting for a update, take your time 👍

**Comment by lionel-bijaoui** [comment]
Just be careful if you use it in production, this is still technically an alpha (v0.x.x) and API may change a lot.
This is why feedback from user like your is very valuable.
You can also spread the word about this plugin.
We need help for accessibility related issue #74 , and we don't have any experience about it. Someone who know about this would be super helpful. Just saying !

**Comment by Cannonb4ll** [comment]
I have been sharing this plugin a lot, actually haha. Didn't knew it was in Alpha! I am planning on using this plugin in a production environment, not a big application but it works just fine to be honest.

I will be looking into your issue as well when I get the time. (#74)

**Comment by lionel-bijaoui** [comment]
Thank you !
Don't worry, every release is working, it just that between release, big change might be introduced. We try to minimized that, but we can't guaranty it.

**Comment by icebob** [comment]
It is alpha, but working :) We have [break changes](https://github.com/icebob/vue-form-generator/releases) in releases. So if you would like to use in production, we recommend to lock the version in package.json and update only manually.

**Comment by lionel-bijaoui** [comment]
@icebob hive mind! 😄

**Comment by icebob** [comment]
Fixed. Will be released in v0.6.0
